src/store/settings/index.ts

```diff
@@ -14,6 +14,8 @@ interface State {
     hasReceivedErrorReportingAlert: boolean;
     enablePlaybackReporting: boolean;
     colorScheme: ColorScheme;
+    sleepTime: number;
+    enabledSleeper: boolean
```

Comment:
I'd prefer a simpler implementation where you set a date at which the timer triggers, rather than tracking the duration, whether it's enabled and how much time is remaining.

Reply:
Should we set a specific interval. For example 30 minutes after time has been triggered.